SPECS-EXTENDED/webkit2gtk3/webkit2gtk3.spec

```diff
@@ -0,0 +1,705 @@
+Requires:       %{name}-jsc-devel%{?_isa} = %{version}-%{release}
+Obsoletes:      webkitgtk4-devel < %{version}-%{release}
+Provides:       webkitgtk4-devel = %{version}-%{release}
+
+%description    devel
+The %{name}-devel package contains libraries, build data, and header
+files for developing applications that use %{name}.
+
+%if %{with docs}
+%package        doc
+Summary:        Documentation files for %{name}
+BuildArch:      noarch
+Requires:       %{name} = %{version}-%{release}
+Obsoletes:      webkitgtk4-doc < %{version}-%{release}
+Provides:       webkitgtk4-doc = %{version}-%{release}
+
+%description    doc
+This package contains developer documentation for %{name}.
+%endif
+
+%package        jsc
+Summary:        JavaScript engine from %{name}
+Obsoletes:      webkitgtk4-jsc < %{version}-%{release}
+Provides:       webkitgtk4-jsc = %{version}-%{release}
+
+%description    jsc
+This package contains JavaScript engine from %{name}.
+
+%package        jsc-devel
+Summary:        Development files for JavaScript engine from %{name}
+Requires:       %{name}-jsc%{?_isa} = %{version}-%{release}
+Obsoletes:      webkitgtk4-jsc-devel < %{version}-%{release}
+Provides:       webkitgtk4-jsc-devel = %{version}-%{release}
+
+%description    jsc-devel
+The %{name}-jsc-devel package contains libraries, build data, and header
+files for developing applications that use JavaScript engine from %{name}.
+
+%prep
+#%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%autosetup -p1 -n webkitgtk-%{version}
+
+# Remove bundled libraries
+rm -rf Source/ThirdParty/gtest/
+rm -rf Source/ThirdParty/qunit/
+
+%build
+# Increase the DIE limit so our debuginfo packages could be size optimized.
+# Decreases the size for x86_64 from ~5G to ~1.1G.
+# https://bugzilla.redhat.com/show_bug.cgi?id=1456261
+%global _dwz_max_die_limit 250000000
+# The _dwz_max_die_limit is being overridden by the arch specific ones from the
+# redhat-rpm-config so we need to set the arch specific ones as well - now it
+# is only needed for x86_64.
+%global _dwz_max_die_limit_x86_64 250000000
+
+# Remove debuginfo to reduce memory consumption:
+# https://bugs.webkit.org/show_bug.cgi?id=140176
+# https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/I6IVNA52TXTBRQLKW45CJ5K4RA4WNGMI/
+%ifarch %{arm} %{ix86}
+%global debug_package %{nil}
+%global optflags %(echo %{optflags} | sed 's/-g /-g0 /')
+%endif
+
+%cmake \
+  -DPORT=GTK \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DENABLE_MINIBROWSER=ON \
+  -DUSE_SOUP2=ON \
+  -DUSE_LD_GOLD=ON \
+  -DUSE_WPE_RENDERER=ON \
+  -DPYTHON_EXECUTABLE=%{_bindir}/python3 \
+  -DENABLE_GTKDOC=ON \
+  -DENABLE_GAMEPAD=OFF \
+  -DUSER_AGENT_BRANDING="Mariner" \
+  %{nil}
+
+%cmake_build -j 4
+
+%install
+%cmake_install
+
+%find_lang WebKit2GTK-4.0
+
+# Finally, copy over and rename various files for %%license inclusion
+%add_to_license_files Source/JavaScriptCore/COPYING.LIB
```

Comment:
As far as I can tell, `%add_to_license_files` isn't defined anywhere- not by Mariner, and not in this spec. It looks like you might have removed the definition while cleaning up the spec- can you add it back in? I see it's defined in the Fedora spec [here](https://sourcegraph.com/src.fedoraproject.org/rpms/webkit2gtk3).